README.md

```diff
@@ -43,9 +43,10 @@ The following details the configurations for the spire agent. The configurations
  |TrustBundlePath        |  Path to trusted CA Cert bundle                                      |
  |TrustDomain            |  SPIFFE trustDomain of the SPIRE Agent                               |
  |JoinToken              |  A join token which has been generated by the server                 |
+ |Umask                  |  Umask to use for new files (if empty then umask is not changed)     |
```

Comment:
Do we want to make this configurable? If so, we should probably communicate a warning - setting this value loosely is extremely dangerous

Reply:
I'm not sure if there is a correct umask value for every situation, that's why it was made configurable.

Reply:
Fair enough... I still feel that we should issue a stern warning... setting this wrong could result in a huge security breach.